app/controllers/guitars_controller.rb

```diff
@@ -25,6 +25,10 @@ def index
       }
     ]
   end
+
+  def guitar_gallery; end
+
+  def welcome_page; end
```

Comment:
at the moment, it just renders the text `Guitars#welcomePage`/`Guitars#guitarGallery` with the React app below.
Are there any further plans for these actions?

Reply:
Do you think that we would need these actions? The future plan is to use React Router and to make it a single page application... I thought we could keep them in case something important pops out in the course concerning the Rails part of the application 
What do you think? @Ron-Lavi